packages/calcite-components/src/components/combobox-item/combobox-item.tsx

```diff
@@ -160,7 +160,7 @@ export class ComboboxItem implements ConditionalSlotComponent, InteractiveCompon
     this.selected = !this.selected;
   }
 
-  itemClickHandler = (event: MouseEvent): void => {
+  itemPointerHandler = (event: PointerEvent): void => {
     event.preventDefault();
```

Comment:
Can you confirm the event still needs to be canceled here? I strongly suspect it is no longer needed because:

1. It was added in a very early stage of the component
2. In this early iteration, it was set on an `<a>` 
3. Not sure why it was canceled in 2 since `href` was set to `#`, which won't redirect – maybe it was an extra step to prevent redirects (canceling `click` stops `<a>`'s default behavior)
4. This PR uses `pointerdown` (vs `click`) and canceling it will prevent `mousedown` and `mouseup` from firing (this was not the case when `click` was canceled)
5. The pointer event is set on `<li>`, which does not have any default click behavior (unlike `<a>`)

If not needed, it would help us avoid adding `pointerEventPreventDefaultOnClick` to the `disabled` test helper options.

Reply:
The reason for using `event.preventDefault` in this scenario is to avoid emitting compatibility `mouse` 

1.  Without preventDefault `mouse` compatibility events will be fired thus causing `blur` on `input` which results in loosing focus.
2.  Preventing mouse  events wouldn't cause any issues since we don't have any listeners or handlers for mouse events in combobox-item and combobox. 
3.  Though `<li>` element does not have any default click behavior , the preventDefault is used to prevent mouse and focus related events.